kairon/chat/routers/channels.py

```
@@ -11,6 +11,16 @@
 router = APIRouter()
 
 
+@router.get("/{bot}/user/posts")
+async def get_user_posts(
+        request: Request,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=CHAT_ACCESS)
```

Ruff (0.12.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)